### PR TITLE
Add GHA setup instructions

### PR DESCRIPTION
**Pull request by therealprof**
Signed-off-by: Daniel Egger <daniel@eggers-club.de>

### CODE REVIEW

File: ops/post-transfer.md [hunk 1]
```diff
@@ -90,6 +90,175 @@ status = ["continuous-integration/travis-ci/push"]
...
+  to the "Settings" tab.
+
+- Synchronize both "Reviewers" and "Members" to people with "Push" (write) access to the
+    repository.
+
+## CI via GitHub Actions (GHA)
+
+### Changes to GitHub repository configuration
+
+No changes are needs as GHA are automatically enabled by default for every organisation.
```
Comment (Disasm) [inline ops/post-transfer.md:103]: "needs"?

Comment (therealprof) [reply ops/post-transfer.md]: Fixed, thanks!

File: ops/post-transfer.md [hunk 2]
```diff
@@ -90,6 +90,175 @@ status = ["continuous-integration/travis-ci/push"]
...
+required_approvals = 1
+status = [
+  "ci-linux (stable, x86_64-unknown-linux-gnu)",
+  "ci-linux (stable, thumbv6m-none-eabi)",
+  "ci-linux (stable, thumbv7m-none-eabi)",
+  "ci-linux (1.35.0, x86_64-unknown-linux-gnu)",
+]
+```
+
+### Bors
```
Comment (adamgreig) [inline ops/post-transfer.md:258]: Is GitHub acting up or have you duplicated this `Bors` section?

File: ops/post-transfer.md [hunk 3]
```diff
@@ -90,6 +90,175 @@ status = ["continuous-integration/travis-ci/push"]
...
+          args: --target=${{ matrix.TARGET }}
+```
+
+
+#### Add a `.github/bors.toml` file with these contents:
+
+The status checks are unfortunately a bit tedious as wildcard on the status can
+**not** be used here, so the information needs to be customized to match the
+information in the `ci.yml`, e.g. for the above snippets the required
+`bors.toml`would look like:
```
Comment (adamgreig) [inline ops/post-transfer.md:244]:
```suggestion
`bors.toml` would look like:
```

File: ops/post-transfer.md (hunk 2 above)
Comment (therealprof) [reply ops/post-transfer.md:258]: There're two bors related sections, but one for bors itself and one for GHA specific bors configuration on different levels. I don't see a duplicated section.

Comment (adamgreig) [reply ops/post-transfer.md:258]: If you view the [new file](https://github.com/rust-embedded/wg/blob/6407da6edc0565da3c101bc4616f1a3e4f814dbb/ops/post-transfer.md) it has two identical Bors sections.

Comment (therealprof) [reply ops/post-transfer.md]: You're right, good spotting. Fixed now.

**Pull request merged**